### ISSUE TITLE
Ownership & Deletion

### ISSUE THREAD

**Issue by Flixbox** [Flixbox/lp-compat#102]
Every item should have an "Owner" / "OP" (original poster) tag.

These users get the ability to remove the app that was posted.

**Comment by 2chiefk** [comment]
Yeah you tried to borrow me from Reddit how do you like this one oh wow I'm even older in this account and like I was trying to I am one of the original Android Engineers yes we swear our brains off okay there's no reason to be borrowing people if you're working on Android software and the software you're looking for I can give you the most current copy because the guy I work with in Russia with his Play Store so it's not even in English we have a copy I could give it to you but you already bugged me on Reddit right so why should I help you here think about it why don't you give me a text back huh don't do that stuff I haven't even on Reddit and how many months I'm the breakfast guy okay and that's a two and a half year old account so you can see I don't go on that often I could help right now but why should I cuz yeah your Lucky Patcher is way out of date

**Comment by 2chiefk** [comment]
Do you have any response for yourself now cuz if you don't I'll just keep it the way it is where it doesn't work for anybody you're lucky patcher that's fine with me peace out I'll give you some minutes here and then I'm going I'll be off read it in the site for quite a long time because I don't pop in that often unless I see some serious issues going on with my operating system I'm going to you know I am Android it's my operating system man designed it when I was a kid
